.github/workflows/ci.yml

```diff
@@ -111,24 +113,22 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Setup Go
-        uses: actions/setup-go@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Golang with cache
+        uses: magnetikonline/action-golang-cache@v3
         with:
           go-version: ${{ env.GO_VERSION }}
-
```

Comment:
Why there are many empty lines deleted? I think we should have an empty line between every two steps.

Reply:
there has no common usage, so jobs are new line seperated,steps will be no new line, or all steps have new line is
 better?